gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -927,7 +939,7 @@ MM_VerboseHandlerOutput::handleConcurrentStart(J9HookInterface** hook, UDATA eve
 	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
 	UDATA contextId = stats->_cycleID;
 	char tagTemplate[200];
-	getTagTemplate(tagTemplate, sizeof(tagTemplate), _manager->getIdAndIncrement(), getConcurrentTypeString(), contextId, omrtime_current_time_millis());
+	getTagTemplate(tagTemplate, sizeof(tagTemplate), _manager->getIdAndIncrement(), getCycleType(stats->_concurrentCycleType), contextId, omrtime_current_time_millis());
```

Comment:
This would break OpenJ9. OpenJ9 will return "default". Not sure it is OK.

Reply:
We can move forward with changes  tolerate the inaccurate "default" string. It won't impact much, the time between these changes making it in and the J9 changes going isn't much.